Subject: [PATCH] AoE warning on 64-bit archs
From: Linux Kernel Mailing List <linux-kernel@vger.kernel.org>
To: bk-commits-head@vger.kernel.org
Patch-mainline: yes

ChangeSet 1.1994.13.1, 2005/03/09 10:20:37-08:00, alexn@dsv.su.se

	[PATCH] AoE warning on 64-bit archs
	
	I just accidently built AoE on x86-64 and it emits a warning
	due to conversion of types of different size, trivial fix:
	
	
	Signed-off-by: Alexander Nyberg <alexn@dsv.su.se>
	Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>



 aoechr.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Signed-off-by: Olaf Hering <olh@suse.de>

diff -Nru a/drivers/block/aoe/aoechr.c b/drivers/block/aoe/aoechr.c
--- a/drivers/block/aoe/aoechr.c	2005-03-09 18:11:26 -08:00
+++ b/drivers/block/aoe/aoechr.c	2005-03-09 18:11:26 -08:00
@@ -178,13 +178,13 @@
 static ssize_t
 aoechr_read(struct file *filp, char __user *buf, size_t cnt, loff_t *off)
 {
-	int n;
+	unsigned long n;
 	char *mp;
 	struct ErrMsg *em;
 	ssize_t len;
 	ulong flags;
 
-	n = (int) filp->private_data;
+	n = (unsigned long) filp->private_data;
 	switch (n) {
 	case MINOR_ERR:
 		spin_lock_irqsave(&emsgs_lock, flags);
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

